static-site/content/SARS-CoV-2-Datasets.yaml

```diff
@@ -1940,3 +2148,609 @@ datasets:
     org:
       name: Center for Medical Genetics, Keio University School of Medicine
       url: https://cmg.med.keio.ac.jp/
+
+  - url: https://nextstrain.org/groups/africa-cdc/ncov/central-africa?c=clade_membership&f_country=Burundi,Cameroon,Central%20African%20Republic,Democratic%20Republic%20of%20the%20Congo,Equatorial%20Guinea,Gabon,Republic%20of%20the%20Congo,Chad&p=grid&tl=country
+    name: Central Africa
+    geo: central-africa
+    level: region
+    coords:
+      - 17.2236
+      - -4.010
+    org:
+      name: Africa CDC PGI
+      url: https://africacdc.org/institutes/ipg/
+
+  - url: https://nextstrain.org/groups/africa-cdc/ncov/eastern-africa?c=clade_membership&f_country=Comoros,Djibouti,Ethiopia,Kenya,Madagascar,Mauritius,Rwanda,Seychelles,Somalia,South%20Sudan,Sudan,Uganda&p=grid&tl=country
+    name: Eastern Africa
+    geo: eastern-africa
+    level: region
+    coords:
+      - 35.993
+      - 9.132
+    org:
+      name: Africa CDC PGI
+      url: https://africacdc.org/institutes/ipg/
+
+  - url: https://nextstrain.org/groups/africa-cdc/ncov/northern-africa?c=clade_membership&f_country=Algeria,Egypt,Libya,Morocco,Tunisia&p=grid&tl=country
+    name: Northern Africa
+    geo: northern-africa
+    level: region
+    coords:
+      - 67.139
+      - 0.788
+    org:
+      name: Africa CDC PGI
+      url: https://africacdc.org/institutes/ipg/
+
+  - url: https://nextstrain.org/groups/africa-cdc/ncov/southern-africa?c=clade_membership&f_country=Angola,Botswana,Eswatini,Lesotho,Malawi,Mozambique,Namibia,South%20Africa,Zambia,Zimbabwe&p=grid&tl=country
+    name: Southern Africa
+    geo: southern-africa
+    level: region
+    coords:
+      - 22.438
+      - -22.322
+    org:
+      name: Africa CDC PGI
+      url: https://africacdc.org/institutes/ipg/
+  
+  - url: https://nextstrain.org/groups/africa-cdc/ncov/western-africa?c=clade_membership&f_country=Benin,Burkina%20Faso,Cabo%20Verde,Cote%20d%27Ivoire,Gambia,Ghana,Guinea,Guinea-Bissau,Liberia,Mali,Niger,Nigeria,Senegal,Sierra%20Leone,Togo&p=grid&tl=country
+    name: Western Africa
+    geo: western-africa
+    level: region
+    coords:
+      - -3.803
+      - 12.243
+    org:
+      name: Africa CDC PGI
+      url: https://africacdc.org/institutes/ipg/
+
+
+  - url: https://nextstrain.org/groups/africa-cdc/ncov/burundi?c=clade_membership&f_country=Burundi&p=grid&tl=country
+    name: Burundi
+    geo: burundi
+    level: country
+    coords:
+      - 29.8870575
+      - -3.3634357
+    org:
+      name: Africa CDC PGI
+      url: https://africacdc.org/institutes/ipg/
+
+  - url: https://nextstrain.org/groups/africa-cdc/ncov/cameroon?c=clade_membership&f_country=Cameroon&p=grid&tl=country
+    name: Cameroon
+    geo: cameroon
+    level: country
+    coords:
+      - 13.1535811
+      - 4.6125522
```

Comment:
@j23414 Can you assign different lat/lng for this Cameroon build so it doesn't overlap with [Bryan Tegomoh's existing Cameroon build](https://github.com/nextstrain/nextstrain.org/blob/a78061708ffb23d15e596354835fe8229329e487/static-site/content/SARS-CoV-2-Datasets.yaml#L517-L526)?

Reply:
Fixed! Sorry I should have checked the [map](https://nextstrain-s-updatelink-p4ai8z.herokuapp.com/sars-cov-2/#datasets) for overlaps